launch.sh

```diff
@@ -10,11 +10,37 @@ until pg_isready; do
 done
```

Comment:
This `pg_isready` needs env vars exported, and will fail if connecting to a separate DB. You will want to test this setup, I think there may be more bugs lurking that I did not see visually.